### DISCUSSION TITLE
Where are posts and pages?

### DISCUSSION THREAD

**Discussion by stoutZero** [area17/twill#1104] [Q&A] [answered]
If twill really is a CMS (toolkit / whatever), where are the posts & pages?

FYI, I have followed the docs until here: ['title' => 'Pages'](https://twill.io/docs/#:~:text=php%0A%0Areturn%20%5B%0A%20%20%20%20%27pages%27%20%3D%3E%20%5B-,%27title%27%20%3D%3E%20%27Pages%27%2C,-%27module%27%20%3D%3E%20true%2C%0A%20%20%20%20%5D%2C%0A%20%20%20%20%27work)

And this is what I got:

![Screenshot at Aug 25 12-36-27](https://user-images.githubusercontent.com/491718/130732500-556a0864-cf58-4054-94f4-8294499a630d.png)

and there aren't any mentions of posts / pages in twill's docs sidebar, so,
are the builtin content supposed to be just
Media + Users?

**Comment by pboivin** [comment, marked as answer]
Hi @stoutZero. That's right, there are no posts and pages out of the box. Unlike other similar tools, Twill makes no assumptions on your actual content. You may need posts, pages, both or none at all, it's really up to you to shape your content with a combination of modules, blocks, form fields, etc.

This starter project may be useful if you are looking for an example of a Pages module:
https://github.com/croustille-io/twill-boilerplate

**Reply by stoutZero** [reply]
Well, ok, I might think some example of how the 'traditional' cms might look like in twill, somewhere in twill's site / docs might be good, that's just me tho.

Thanks for the reference to that boilerplate.

Cheers.

<small>P.S.: I can't mark this as answer, GH throwing error 422...</small>
